actors/system/src/lib.rs

```diff
@@ -25,9 +25,25 @@ pub enum Method {
 /// System actor state.
 #[derive(Default, Deserialize, Serialize)]
 #[serde(transparent)]
-pub struct State([(); 0]);
+pub struct State {
+    // builtin actor registry: Vec<(String, Cid)>
+    builtin_actors: Cid,
+}
 impl Cbor for State {}
 
+impl State {
+    pub fn get_builtin_actors<B: Blockstore>(
+        &self,
+        store: &B,
+    ) -> Result<Vec<(String, Cid)>, String> {
```

Comment:
```suggestion
    ) -> anyhow::Result<Vec<(String, Cid)>> {
```

(makes the code below simpler)

Reply:
we don't use anyhow in the builtin-actors code as far as I can tell, so I will decline this suggestion for consistency with the rest of the code again.

Note that it is not a bad suggestion at all, and I would definitely apply it in ref-fvm code, just not in builtin-actors.

Reply:
Fair enough. Although it would be nice to not use strings...